anoncreds/Cargo.toml

```diff
@@ -19,29 +19,29 @@ crate-type = ["staticlib", "rlib", "cdylib"]
 default = ["ffi"]
 ffi = ["ffi-support", "logger", "zeroize"]
 logger = ["env_logger"]
-vendored = ["indy-data-types/vendored"]
+vendored = ["openssl", "openssl/vendored"]
+merkle_tree = ["indy-utils/hash", "hex"]
 
 [dependencies]
-env_logger = { version = "0.7.1", optional = true }
+env_logger = { version = "0.9.3", optional = true }
 ffi-support = { version = "0.4.0", optional = true }
 log = "0.4"
 once_cell = "1.9"
-rand = "0.7"
+rand = "0.8.5"
 regex = "1.2.1"
 serde = { version = "1.0", features = ["derive"] }
-serde_json = "1.0"
-sha2 = "0.9"
+serde_json = { version = "1.0", features = ["raw_value"]}
+sha2 = "0.10"
 tempfile = "3.1.0"
 thiserror = "1.0"
-zeroize = { version = "1.3", optional = true }
+ursa = { version = "=0.3.6", default-features = false, features = ["cl_native", "serde"] }
+zeroize = { version = "1.3", optional = true, features = ["zeroize_derive"] }
+hex = { version = "0.4", optional = true }
+openssl = { version = "0.10", optional = true }
```

Comment:
Where are openssl and hex used?

Reply:
Only in copied features vendored and merkle_tree.

Reply:
Yeah would be good if we can remove the openssl dependency (we still pull it in via `Ursa` though) as it can cause quite some issues when building for different platforms.